INSTALL.md

```diff
@@ -82,10 +82,10 @@ If the user is not root and should use "su" to become root, use the parameter `b
 
 * `host_vars/TARGET_NAME` this is a file that you must create
 
-For example, you can create the file  `host_vars/myserver.mydomain.local` with the following content:
+For example, you can create the file  `host_vars/server.mydomain.local` with the following content:
 
 ```
-hostname: backend.mydomain.local
+hostname: server.mydomain.local
```

Comment:
I think that it needs to be clarified that TARGET_NAME is supposed to be assigned to "hostname" inside of this file

Reply:
The `host_vars/TARGET_NAME` file can be anything, that `TARGET_NAME` is arbitrary but needs to be listed in the `hosts` file.

Then yes, the `hostname` value defined inside the `host_vars/TARGET_NAME` file does need to be an actual hostname.  But that doesn't need to match `TARGET_NAME`.

In the examples, it's probably best to use an actual example name everywhere i.e. `host_vars/local` to avoid confusionn.